packages/labs/examples/popover2Example.tsx

```diff
@@ -40,10 +40,10 @@ const POPPER_DOCS = "https://popper.js.org/popper-documentation.html#modifiers";
 export interface IPopover2ExampleState {
     canEscapeKeyClose?: boolean;
     exampleIndex?: number;
+    hasBackdrop?: boolean;
     inheritDarkTheme?: boolean;
     inline?: boolean;
     interactionKind?: PopoverInteractionKind;
-    isModal?: boolean;
```

Comment:
Mostly consistency with `Overlay`. Happy to change `hasBackdrop` to something better later